src/main/kotlin/lotto/domain/LottoesRank.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+class LottoesRank(
+    private val value: Map<Rank, List<LottoTicket>>
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private val value: Map<Rank, Int>
```